test/spec/faros_event_community_edition_spec.sh

```diff
@@ -1,5 +1,6 @@
 Describe 'faros_event.sh (Community edition)'
   export FAROS_NO_FORMAT=1
+  export FAROS_DRY_RUN=1
```

Comment:
I might have missed the reasoning for removing this from the tests. Seems simpler to just always make sure the test have dry run set.

Reply:
I made that change as part of https://github.com/faros-ai/faros-events-cli/pull/68 because I wanted to be able to run the tests and hit the actual hasura endpoints. After discussion with @tovbinm , we agreed to do this kind of testing in the Community edition repo instead. So feel free to revert (the GHA and the README needs updating to undo passing `FAROS_DRY_RUN`).

Reply:
Yep, let's revert.